resources/lang/en-GB/general.php

```diff
@@ -88,6 +88,9 @@
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
     'first'					=> 'First',
+    'firstnamelastname'     => 'First Name Last Name (janesmith@example.com)',
+    'lastname_firstinitial' => 'Last Name First Initial (smith_f@example.com)',
```

Comment:
This should probably be `smith_j@example.com`, no?

Reply:
typo has been corrected.